### ISSUE TITLE
Add link to reference and footnote section to table of content

### ISSUE THREAD

**Issue by asmaier** [arxiv-vanity/engrafo#269]
I'm not sure if that is possible within the Distill HTML output, but it would be nice to have a link to the reference and footnote section in the table of content.

**Comment by bfirsh** [comment]
Which table of contents do you mean? Depends on #101 I think. :)
